ddln_imaginator.proto

```diff
@@ -63,8 +60,9 @@ message Environment {
         // density value, typical range 0..1 [-]
         optional float density = 3;
 
-        // directional x, y, z values to set the wind in ENU coordinates system [m/s]
-        optional Point3d velocity = 4;
+        // directional x, y values for displacement in ENU coordinates system [m] and z value for shape changing
```

Comment:
If possible, describe unit for z as well.

Reply:
It will be all in meters.